src/main/java/com/ngs/stash/externalhooks/hook/ExternalMergeCheckHook.java

```diff
@@ -137,44 +127,34 @@ public void check(
         String summaryMsg = "Merge request failed";
 
         try {
-            int Result = runExternalHooks(pb, refChanges, hookResponse);
-            if (Result != 0) {
-                String rawDetailedMsg = errWriter.toString();
-                String prePrefix = "<pre style=\"overflow: auto; white-space: nowrap;\">";
-                String preSuffix = "</pre>";
-                String detailedMsg = prePrefix + rawDetailedMsg.replaceAll("(\r\n|\n)", "<br/>").replaceAll(" ", " ") + preSuffix;
```

Comment:
(Nevermind; I see from #53 that dropping this is a good thing to do. It sounds like @seletskiy has already made the matching change to his version of the plugin.)